test/unit/spec/common/jobCommChannel-spec.js

```diff
@@ -243,19 +447,26 @@ define([
                     .initCommChannel()
                     .then(() => {
                         expect(comm.comm).not.toBeNull();
+                        spyOn(comm, 'sendCommMessage').and.callFake((...args) => {
+                            return comm.sendCommMessage.and.originalFn.call(comm, ...args);
+                        });
                         return new Promise((resolve) => {
                             // resolve the promise when we see comm.comm.send(...)
-                            spyOn(comm.comm, 'send').and.callFake((...args) => {
-                                resolve(args);
+                            spyOn(comm.comm, 'send').and.callFake(() => {
+                                resolve();
                             });
                             testBus.emit(testCase.channel, testCase.message);
                         });
                     })
-                    .then((args) => {
+                    .then(() => {
                         expect(comm.comm.send).toHaveBeenCalled();
-                        expect(args[0]).toEqual(
-                            Object.assign(testCase.expected, { target_name: 'KBaseJobs' })
-                        );
+                        expect(comm.comm.send.calls.allArgs()).toEqual([
+                            [Object.assign(testCase.expected, { target_name: 'KBaseJobs' })],
+                        ]);
```

Comment:
Pretty much the same as the old test, just ensures that there has only been one call.